### PR TITLE
OCPBUGS-15034 correcting minor misnaming and adding optional

### PR DESCRIPTION
**Pull request by kquinn1204**
[OCPBUGS-15034]: Issue in file networking/multiple_networks/configuring-additional-network.adoc

Version(s): 4.10 and greater including main

Issue: https://issues.redhat.com/browse/OCPBUGS-15034

Link to docs preview: https://61277--docspreview.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network.html#nw-multus-vlan-object_configuring-additional-network

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by kquinn1204** [comment]
@mlguerrero12 Please review this clean up update adding optional and ensuring names inline with what is specified in the extract.

File: modules/nw-multus-bridge-object.adoc [hunk 1]
```diff
@@ -23,50 +23,78 @@ plugin:
 
 |`type`
 |`string`
-|
+|`bridge`
```
Comment (mlguerrero12) [inline modules/nw-multus-bridge-object.adoc:26]: this is duplicated?

File: modules/nw-multus-bridge-object.adoc [hunk 2]
```diff
@@ -23,50 +23,78 @@ plugin:
...
-|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
+|`enabledad`
+|`boolean`
+|Optional: Enables duplicate address detection for the container side `veth`. The default value is `false`.
+
+|`macspoofchk`
+|`boolean`
+|Optional: Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. The default value is `false`.
```
Comment (mlguerrero12) [inline modules/nw-multus-bridge-object.adoc:78]: you are missing the param preserveDefaultVlan

File: modules/nw-multus-host-device-object.adoc [hunk 3]
```diff
@@ -44,7 +43,7 @@ The following object describes the configuration parameters for the host-device
 |Optional: The Linux kernel device path, such as `/sys/devices/pci0000:00/0000:00:1f.6`.
 
 |`pciBusID`
-|`string`
+|`string` (optional)
```
Comment (mlguerrero12) [inline modules/nw-multus-host-device-object.adoc:46]: why optional here?

File: modules/nw-multus-ipvlan-object.adoc [hunk 4]
```diff
@@ -26,23 +26,23 @@ plugin:
 
 |`type`
 |`string`
-|The name of the CNI plugin to configure: `ipvlan`.
+|Optional: The name of the CNI plugin to configure: `ipvlan`.
```
Comment (mlguerrero12) [inline modules/nw-multus-ipvlan-object.adoc:29]: the type is not optional

File: modules/nw-multus-ipvlan-object.adoc [hunk 5]
```diff
@@ -26,23 +26,23 @@ plugin:
...
-|Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
 |`ipam`
-|`object`
-|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition.
+|`object` (required unless chained)
+|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition. This required unless chained.
 
 Do not specify `dhcp`. Configuring IPVLAN with DHCP is not supported because IPVLAN interfaces share the MAC address with the host interface.
```
Comment (mlguerrero12) [inline modules/nw-multus-ipvlan-object.adoc:47]: I'm missing the notes to understand what chained means. It's in the official doc of cni

File: modules/nw-multus-bridge-object.adoc (hunk 1 above)
Comment (kquinn1204) [reply modules/nw-multus-bridge-object.adoc:26]: @mlguerrero12 I am taking this from https://www.cni.dev/plugins/current/main/bridge/ `type (string, required): “bridge”.`

File: modules/nw-multus-bridge-object.adoc (hunk 2 above)
Comment (kquinn1204) [reply modules/nw-multus-bridge-object.adoc:78]: added

File: modules/nw-multus-host-device-object.adoc (hunk 3 above)
Comment (kquinn1204) [reply modules/nw-multus-host-device-object.adoc:46]: That is a mistake

File: modules/nw-multus-ipvlan-object.adoc (hunk 5 above)
Comment (kquinn1204) [reply modules/nw-multus-ipvlan-object.adoc:47]: I added a note on this

Comment (mlguerrero12) [reply modules/nw-multus-ipvlan-object.adoc:47]: I meant this notes for ipvlan https://www.cni.dev/plugins/current/main/ipvlan/#notes. The last bullet is about the ipam.

File: modules/nw-multus-bridge-object.adoc (hunk 1 above)
Comment (kquinn1204) [reply modules/nw-multus-bridge-object.adoc:26]: I removed bridge to avoid any confusion

Comment (mlguerrero12) [reply modules/nw-multus-bridge-object.adoc:26]: it should be 

"The name of the CNI plugin to configure: bridge."

like we have in the other plugins

Comment (kquinn1204) [reply modules/nw-multus-bridge-object.adoc]: Yes just added that for consistency

**Comment by kquinn1204** [comment]
@mlguerrero12 Can you add a LGTM if happy with updates on this PR.

File: modules/nw-multus-ipvlan-object.adoc (hunk 5 above)
Comment (mlguerrero12) [reply modules/nw-multus-ipvlan-object.adoc]: did you address this? I still see the previous note about chained plugins.

File: modules/nw-multus-ipvlan-object.adoc [hunk 6]
```diff
@@ -28,26 +28,31 @@ plugin:
...
-|The configuration object for the IPAM CNI plugin. The plugin manages IP address assignment for the attachment definition.
-
-Do not specify `dhcp`. Configuring IPVLAN with DHCP is not supported because IPVLAN interfaces share the MAC address with the host interface.
+|Optional: Set the maximum transmission unit (MTU) to the specified value. The default value is automatically set by the kernel.
 
 |====
 
+[NOTE]
+====
+* `ipvlan` does not allow virtual interfaces to communicate with the `master` interface. Therefore the container will not be able to reach the host by using the `ipvlan` interface. Be sure that the container joins a network that provides connectivity to the host, such as a network supporting the Precision Time Protocol (`PTP`).
```
Comment (skrthomas) [inline modules/nw-multus-ipvlan-object.adoc:51]:
```suggestion
* The `ipvlan` object does not allow virtual interfaces to communicate with the `master` interface. Therefore the container will not be able to reach the host by using the `ipvlan` interface. Be sure that the container joins a network that provides connectivity to the host, such as a network supporting the Precision Time Protocol (`PTP`).
```

File: modules/nw-multus-ipvlan-object.adoc [hunk 7]
```diff
@@ -28,26 +28,31 @@ plugin:
...
+* A single `master` interface cannot be enslaved by both `macvlan` and `ipvlan`.
```
Comment (skrthomas) [inline modules/nw-multus-ipvlan-object.adoc:52]: I'm guessing `master` `enslaved` usage here is unavoidable, given: https://redhat-documentation.github.io/supplementary-style-guide/#_master_and_slave

**Review by skrthomas — COMMENTED** [review]
A couple of comments for you to consider. Looks good!

File: modules/nw-multus-ipvlan-object.adoc (hunk 7 above)
Comment (kquinn1204) [reply modules/nw-multus-ipvlan-object.adoc:52]: I took this directly from https://www.cni.dev/plugins/current/main/ipvlan/ but to be honest I am uncomfortable with terminology .  I changed, and actually I understand it better with the update.

**Pull request merged**